internal/provider/node_room_member.go

```diff
@@ -62,14 +80,76 @@ the Netdata child agents across different rooms because by default all of them e
 				},
 			},
 			"node_names": schema.ListAttribute{
-				Description: "List of node names to add to the room",
+				Description: "List of node names to add to the room. At least one node name is required.",
```

Comment:
So I can't just use dynamic rules?

Reply:
below the description there is a 
`Optional:    true`
so when you use it you have to provide at least one element

Reply:
ok now I understand, thx